File: src/main/java/no/nav/pensjon/selvbetjeningopptjening/model/AfpHistorikkDto.java
package no.nav.pensjon.selvbetjeningopptjening.model;

import java.time.LocalDate;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class AfpHistorikkDto {

    LocalDate virkFom;
    LocalDate virkTom;

    public LocalDate getVirkFom() {
        return virkFom;
    }

    public void setVirkFom(LocalDate virkFom) {
        this.virkFom = virkFom;
    }

    public LocalDate getVirkTom() {
        return virkTom;
    }

    public void setVirkTom(LocalDate virkTom) {
        this.virkTom = virkTom;
    }
}

File: src/main/java/no/nav/pensjon/selvbetjeningopptjening/model/AfpHistorikkDto.kt
package no.nav.pensjon.selvbetjeningopptjening.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.time.LocalDate

@JsonIgnoreProperties(ignoreUnknown = true)
data class AfpHistorikkDto (
    val virkFom: LocalDate,
    val virkTom: LocalDate?
)